lua/pixelui/elements/cl_category.lua

```diff
@@ -39,7 +39,7 @@ function PANEL:Paint(w, h)
     self.ArrowRotation = lerp(FrameTime() * 10, self.ArrowRotation, self:GetParent():GetExpanded() and 0 or 90)
 
     local arrowSize = h * .45
-    PIXEL.DrawImgurRotated(w - h * .3 - PIXEL.Scale(4), h / 2, arrowSize, arrowSize, self.ArrowRotation, "30Bvuwi", PIXEL.Colors.PrimaryText)
+    PIXEL.DrawImageRotated(w - h * .3 - PIXEL.Scale(4), h / 2, arrowSize, arrowSize, self.ArrowRotation, "https://cdn.lythium.dev/u/37hCJY.png", PIXEL.Colors.PrimaryText)
```

Comment:
Use a different CDN that will be more reliable, this applies for all images in this PR